### ISSUE TITLE
"0 materials affected" when attemping to swap texture packs

### ISSUE THREAD

**Issue by FouriusVixen** [TheDuckCow/MCprep#82]
The "Swap Texture Pack" button doesn't seem to work.

Tried it out with 3 texture packs; *Faithful++1.14.x+v1.2*, *Stratum-2048x*, and *SapixCraft 256x MC1.14*. Always get a "0 materials affected" message. 

![a6ecb7a9aa6570a194c7fff4812c5246](https://user-images.githubusercontent.com/15067179/63318214-c16cab80-c2e3-11e9-8830-2efaddf353d3.gif)

**Comment by TheDuckCow** [comment]
Hi there, I think from your screenshot you are correctly running this when you have the world selected in the 3D viewport (but double check that you are). My next debugging suggestion is, click farther into the texturepack folder, down into and inside of the block/blocks folder. I believe I have an update pending where I need to treat newer texturepack formats to support a change in subfolder name from "block" to also search for "blocks" (or maybe the other way around). 

So, again at worst, click down into the root folder that has all the actual textures inside the resource pack. Let me know if that works for you!

**Comment by FouriusVixen** [comment]
Hey! I tried out your suggestions; it seemed to only change 12/49 of the textures. 


[(Video was too large for GitHub.)](https://i.imgur.com/6uCOKZH.mp4)

**Comment by FouriusVixen** [comment]
Hey! I tried out your suggestions, but it seemed to only change 12/49 textures.
![2019-08-2423-33-32](https://user-images.githubusercontent.com/15067179/63645251-eab47f80-c6c7-11e9-8a02-b84369c12d0d.gif)

**Comment by Voxolotl** [comment]
That's because 1.13 changed a lot of the internal block names used in resource packs. Because MCprep doesn't recognize those changes yet, any post-1.12 resource pack will not work properly.

**Comment by Voxolotl** [comment]
Yeah, there were some changes with the Principled shader node in Blender 2.8, causing a bug where the normal map feeds into the emission input rather than the normal one. I already reported the issue, should be fixed in the next update, hopefully!
